common/cgroup/cgroup.go

```diff
@@ -252,6 +253,37 @@ func CreateCgroups(root string, subcgroup string, serviceNames []string) error {
 		// directories to mount read-write
 		subcgroupPath := filepath.Join(cPath, subcgroup)
 
+		// Workaround for https://github.com/coreos/rkt/issues/1210
+		// If /machine.slice does not exist, create it and configure it
+		cgroupPathFix := filepath.Join(cPath, "system.slice")
+		if err := os.MkdirAll(cgroupPathFix, 0755); err != nil {
+			return err
+		}
+		knobs := []string{"cpuset.mems", "cpuset.cpus"}
+		for _, knob := range knobs {
+			parentFile := filepath.Join(filepath.Dir(cgroupPathFix), knob)
+			childFile := filepath.Join(cgroupPathFix, knob)
+
+			data, err := ioutil.ReadFile(childFile)
+			// If the file does not exist, just skip it. It will not exist in all
+			// hierarchies.
+			if err != nil {
+				continue
+			}
+			// If the file is already configured, don't change it
+			if strings.TrimSpace(string(data)) != "" {
+				continue
+			}
+
+			data, err = ioutil.ReadFile(parentFile)
+			if err == nil {
```

Comment:
I think we should just ignore the error in that case. I don't see how `err != nil` can happen but this patch is only a workaround, so if we can't do the workaround, let's just continue, it does not worth a panic or printing a warning message IMO. Would a comment about that in the code be good enough?